src/ienv.cpp

```diff
@@ -7,24 +7,27 @@
 #include <irods/rodsClient.h>
 #include <irods/parseCommandLine.h>
 
+#include <cstdlib>
+#include <iostream>
+
 void usage();
 
 int
 main( int argc, char **argv ) {
     signal( SIGPIPE, SIG_IGN );
 
-    char* optStr = "h";
     rodsArguments_t myRodsArgs;
-    int status = parseCmdLineOpt( argc, argv, optStr, 0, &myRodsArgs );
+
+    int status = parseCmdLineOpt(argc, argv, "h", 0, &myRodsArgs);
 
     if ( status < 0 ) {
-        printf( "Use -h for help\n" );
-        exit( 1 );
+        std::cout << "Use -h for help" << std::endl;
```

Comment:
What is the reason for using `std::endl` instead of just `\n` here and throughout this change? Mostly just curious...

Reply:
`std::endl` forces a flush.

Reply:
Okay. Didn't seem strictly necessary from my understanding but it's not a problem, either. Resolving.